test/issues/general/test_5660.test

```diff
@@ -0,0 +1,27 @@
+# name: test/issues/general/test_5660.test
+# description: Issue 5660: Group by all throws binder error when a more informative error should be produced
+# group: [general]
+
+statement ok
+CREATE TABLE foo AS SELECT 'a, b, c' AS "x", '1' AS y;
```

Comment:
Is this really OK that `x` alias is `"`-quoted? E.g. Spark SQL throws a syntax error. Just curious.

Reply:
It is defined in the ANSI SQL standard that identifiers can be quoted with `"`, see [here](https://www.contrib.andrew.cmu.edu/~shadow/sql/sql1992.txt):

```
         <delimited identifier> ::=
              <double quote> <delimited identifier body> <double quote>
```

Some other systems use different quoting rules such as using backticks - but we follow Postgres and the SQL standard here.